lib/commands.py

```diff
@@ -103,9 +107,10 @@ def _ssh(hostname_or_ip, cmd, check, simple_output, suppress_fingerprint_warning
     )
     logging.debug(f"[{hostname_or_ip}] {command}")
     if windows_background:
-        return True, process
+        return process
```

Comment:
This changes the return type, which is not the topic of this commit :sweat_smile: 
It could be better to have a separate commit for this (likely first, because we probably don't want to describe that.

Reply:
it is linked to this commit, because I had to check the type of the returned object instead of relying on the bool value that was also returned. The bool was not used anymore because of the changes for the type hints, so I dropped it.